### DISCUSSION TITLE
Getting a specific page number

### DISCUSSION THREAD

**Discussion by adamhp** [tweepy/tweepy#1883] [Questions]
Hey all, I'm trying to split my requests for specific pages of the `get_followers`. I thought I could maybe use a Cursor manually to get specific pages, but I really haven't been able to figure out. 

Let's say someone has 1000 followers. I'm splitting the jobs into 5 consecutive calls of:
```python
api.get_followers(screen_name=screen_name, count=200)
```

So, I sort of expected the following to work:
```python
api.get_followers(screen_name=screen_name, cursor=-1, count=200)
api.get_followers(screen_name=screen_name, cursor=0, count=200)
api.get_followers(screen_name=screen_name, cursor=1, count=200)
api.get_followers(screen_name=screen_name, cursor=2, count=200)
api.get_followers(screen_name=screen_name, cursor=3, count=200)
```

This way, if any individual job fails, I can pick up where I left off, and I can handle the rate limiting myself with job scheduling.

How can I get a specific page of 200 results from an API call?

**Comment by Harmon758** [comment]
While the first cursor will be `-1`, subsequent cursors returned as `next_cursor` will be ID values, not increments of 1.

https://developer.twitter.com/en/docs/twitter-api/v1/accounts-and-users/follow-search-get-users/api-reference/get-followers-list
https://developer.twitter.com/en/docs/twitter-api/v1/pagination

The `CursorIterator` returned by `Cursor.pages` when used with this method has a `next_cursor` attribute you can use and pass as `cursor`.
The iterator can also be retrieved using the `iterator` attribute of `Cursor` or the `page_iterator` attribute of the `ItemIterator` returned by `Cursor.items`.

**Reply by adamhp** [reply]
Thanks for the reply. I'll see if I can get this to work.

**Reply by adamhp** [reply]
@Harmon758 I'm still not sure this really solves the problem.

If I am making a request for 200 pages, and my process errors out on the 179th iteration, how do I pick up at 180, if I have called .pages(), for example.

**Reply by Harmon758** [reply]
You can pass the `next_cursor` attribute it stops at as the `cursor` parameter for `API.get_followers`.
